### PR TITLE
Updated AllProducts functionality (edit, delete) && created Admin Tools && created productForm && some basic css

### PR DESCRIPTION
**Pull request by rome-777**

### CODE REVIEW

File: client/components/Admin/productForm.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            ImgUrl: product.id ? product.ImgUrl : '',
+        }
+        this.onChange = this.onChange.bind(this);
+        this.handleDestroy = this.handleDestroy.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+    componentDidMount() {
+        // left here to console.log things, any other future uses, etc
+    }
+    // Not sure if we need this, but leaving here for now
```
Comment (kestrelgorlick) [inline client/components/Admin/productForm.js:23]: Please remove

File: client/components/Admin/productForm.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                        }
+                    </div>
+                </form>
+            </div>
+        );
+    }
+}
+
+const mapStateToProps = ({ products }, ownProps) => {
+    // Q: should we rename products (above) to productsReducer or something? Might be a little confusing otherwise
```
Comment (kestrelgorlick) [inline client/components/Admin/productForm.js:104]: Remove

File: client/components/AllProducts.js [hunk 3]
```diff
@@ -1,42 +1,69 @@
...
-        const { products } = this.props;
-
+        const { products, isLoggedIn } = this.props;
         return (
             <div>
-                <ul>
+                <ul id='all-products-list'>
                     {
-                        products.map( (product) => {
+                        products.length !==0 ? products.map((product) => {
```
Comment (kestrelgorlick) [inline client/components/AllProducts.js:22]: Does not need this logic `products.length !==0` etc.

File: client/components/navbar.js [hunk 4]
```diff
@@ -5,22 +5,27 @@ import {logout} from '../store'
 
 const Navbar = ({handleClick, isLoggedIn}) => (
   <div>
-    <h1>FS-App-Template</h1>
+    <h1>Bingr</h1>
```
Comment (kestrelgorlick) [inline client/components/navbar.js:8]: Fix typo

File: client/routes.js [hunk 5]
```diff
@@ -14,20 +13,25 @@ class Routes extends Component {
   }
 
   render() {
-    const {isLoggedIn} = this.props
+    const { isLoggedIn } = this.props
+    //console.log(this.props);
```
Comment (kestrelgorlick) [inline client/routes.js:17]: Remove

File: client/routes.js [hunk 6]
```diff
@@ -14,20 +13,25 @@ class Routes extends Component {
...
+            <Route path='/products/update/:id' component={productForm} />
           </Switch>
         ) : (
           <Switch>
-            <Route path="/login" component={Login} />
-            <Route path="/signup" component={Signup} />
-            <Redirect to="/login" />
+            <Route path='/login' component={Login} />
+            <Route path='/signup' component={Signup} />
+            <Route path='/home' component={Home} />
```
Comment (kestrelgorlick) [inline client/routes.js:33]: Add comment here that we will be displaying home and products for non logged-in users, and that we need to add the functionality in order to tell a user to log in if they try to create an order

File: server/api/products.js [hunk 7]
```diff
@@ -55,4 +55,6 @@ router.delete('/delete/:id',async(req,res,next)=>{
   catch(error){
       next(error)
   }
-})
\ No newline at end of file
+})
+
+// Q: why do we need 'delete' in the route address?
```
Comment (kestrelgorlick) [inline server/api/products.js:60]: Remove

File: server/db/index.js [hunk 8]
```diff
@@ -28,10 +28,14 @@ const syncAndSeed =  async()=> {
   ])
 
   const products = await Promise.all([
-    Product.create({name: 'milk', price: 4, category: 'diary', quantity: 10, descripton: 'milk'})
+    Product.create({ name: 'milk', price: 3, category: 'dairy', quantity: 10, description: 'milk', ImgUrl: 'http://cdn.shopify.com/s/files/1/0071/5826/7994/products/19064664101_800x.jpg?v=1564927226'}),
```
Comment (kestrelgorlick) [inline server/db/index.js:31]: Remove all new product creation because of possible conflict with Karan's merge

File: server/db/index.js [hunk 9]
```diff
@@ -28,10 +28,14 @@ const syncAndSeed =  async()=> {
...
+    Product.create({ name: 'tesla', price: 30000, category: 'car', quantity: 10, description: 'model 3', ImgUrl: 'https://i1.wp.com/eastwest.thegadgetman.org.uk/wp-content/uploads/2017/07/tesla256.png?fit=256%2C256&ssl=1'}),
+    Product.create({ name: 'doge', price: 1, category: 'crypto', quantity: 10, description: 'moon', ImgUrl: 'https://i.pinimg.com/originals/b7/26/7a/b7267aef163f639140791606f0bcdfd6.png' })
   ])
+
   const [cody, murphy] = users;
-  const [milk] = products;
+  const [milk, tesla, doge] = products;
```
Comment (kestrelgorlick) [inline server/db/index.js:37]: Same thing here

File: server/db/models/user.js [hunk 10]
```diff
@@ -18,12 +18,15 @@ const User = db.define('user', {
   githubId: {
     type: Sequelize.INTEGER
   },
-  isAdmin:{
-    type: Sequelize.BOOLEAN
-  },
-  isAuth:{
-    type: Sequelize.BOOLEAN
-  }
+  // name: {
```
Comment (kestrelgorlick) [inline server/db/models/user.js:21]: Remove all model changes, I will update the models to have default values and handle null or empty string edge cases

**Review by kestrelgorlick — CHANGES_REQUESTED** [review]
Great work, once my comments are addressed, you are good to go!

**Comment by rome-777** [comment]
all done

**Pull request closed without merging**